scripts/dev/aliases.sh

```diff
@@ -2,7 +2,7 @@
 
 alias lightning-cli="\$FM_LIGHTNING_CLI"
 alias lncli="\$FM_LNCLI"
-alias bitcoin-cli="\$FM_BTC_CLIENT"
+alias bitcoin-cli="\$FM_BTC_CLIENT -datadir=\$FM_BTC_DIR"
```

Comment:
Shouldn't this be done here? 

https://github.com/fedimint/fedimint/blob/c05926868fc5e983637a25ca879b8e603e055b9d/devimint/src/vars.rs#L137

Reply:
done